code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUser.java

```diff
@@ -62,14 +70,14 @@ public boolean userCanStillSendEmails() {
     return emailsSent < Constants.EMAILING_THRESHOLD;
   }
 
-  public boolean userCanMakeMoreDraftsAfterBeingFlagged() {
-    if (attemptedEmailDrafts > 0) {
-      attemptedEmailDrafts--;
-      return true;
-    } else {
-      return false;
-    }
-  }
+  // public boolean userCanMakeMoreDraftsAfterBeingFlagged() {
```

Comment:
Make sure that this is either completed or removed at the point of the PR. I think it should be removed form this specific one though as I think it's not necessary for what we're discussing here. Remember, keep PRs small.